app/_src/policies/applying-policies.md

```diff
@@ -2,29 +2,84 @@
 title: Applying Policies
 ---
 
-Once installed, {{site.mesh_product_name}} can be configured via its policies. You can apply policies with [`kumactl`](/docs/{{ page.version }}/explore/cli) on Universal, and with `kubectl` on Kubernetes. Regardless of what environment you use, you can always read the latest {{site.mesh_product_name}} state with [`kumactl`](/docs/{{ page.version }}/explore/cli) on both environments.
+Once installed, {{site.mesh_product_name}} can be configured via its policies. 
+You can apply policies with [`kumactl`](/docs/{{ page.version }}/explore/cli) on Universal, and with `kubectl` on Kubernetes. 
+Regardless of what environment you use, you can always read the latest {{site.mesh_product_name}} state with [`kumactl`](/docs/{{ page.version }}/explore/cli) on both environments.
 
 {% tip %}
 We follow the best practices. You should always change your Kubernetes state with CRDs, that's why {{site.mesh_product_name}} disables `kumactl apply [..]` when running in K8s environments.
 {% endtip %}
 
-These policies can be applied either by file via the `kumactl apply -f [path]` or `kubectl apply -f [path]` syntax, or by using the following command:
+{% tabs policy-apply useUrlFragment=false %}
+{% tab policy-apply Kubernetes %}
 
 ```sh
 echo "
-  type: ..
+  apiVersion: kuma.io/v1alpha1
+  kind: ..
   spec: ..
-" | kumactl apply -f -
+" | kubectl apply -f -
 ```
 
-or - on Kubernetes - by using the equivalent:
+{% endtab %}
+{% tab policy-apply Universal %}
 
 ```sh
 echo "
-  apiVersion: kuma.io/v1alpha1
-  kind: ..
+  type: ..
   spec: ..
-" | kubectl apply -f -
+" | kumactl apply -f -
 ```
 
-Below you can find the policies that {{site.mesh_product_name}} supports. In addition to [`kumactl`](/docs/{{ page.version }}/explore/cli), you can also retrieve the state via the {{site.mesh_product_name}} [HTTP API](/docs/{{ page.version }}/reference/http-api) as well.
+{% endtab %}
+{% endtabs %}
+
+In addition to [`kumactl`](/docs/{{ page.version }}/explore/cli), you can also retrieve the state via the {{site.mesh_product_name}} [HTTP API](/docs/{{ page.version }}/reference/http-api) as well.
+
+{% if_version gte:2.6.x %}
+
+## Applying policies on Zone and Global Control Planes
+
+Multi-zone deployment consists of Global Control Plane (Global CP) deployment with one or many Zone Control Planes (Zone CP) connected to it.
+Each Zone CP represents a single cluster (i.e. Kubernetes or Universal cluster). 
+Policies can be applied on both Global and Zone CPs.
+
+When policy is applied on Global CP:
+* it is propagated to all Zone CPs and applied to all matched data plane proxies in all zones
+* Global CP is a source of truth for the policy (when Global CP is down it's not possible to create/update policies)
+
+When policy is applied on Zone CP:
+* it is applied only to matched data plane proxies in the same zone
```

Comment:
I'd put `same zone` in bold.

Should you also update: https://deploy-preview-1591--kuma.netlify.app/docs/dev/production/deployment/multi-zone/#global-control-plane-offline about policy updates ?

Btw if my global is down can I replace and existing policy with one with the same name but with the origin label set?